schema/schema.yaml

```diff
@@ -77,6 +83,7 @@ tables:
       datatype: string
       length: 30
       description: Publication reference; links to Publications table
+      ivoa:ucd: meta.ref
 
     constraints:
```

Comment:
Do we want to explicitly add an index to specify the primary key telescope? This type of information is not consistent across our tables, sometimes we have it and sometimes we don't. The lack of it means it doesn't appear in the documentation even though the primary key should still work in the database.

Reply:
Done.  (Copied your work from SIMPLE.)